### PR TITLE
Feature engagement

### PR DESCRIPTION
**Pull request by patrickmorgan2**
Creation and updating of several pages: Engagement, Discovery, Transition, Deploy, Evaluation and Adoption, FAQ and case studies.

### CODE REVIEW

File: program/engagement/transition.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+## Tools, Templates and Guidance 
+
+* [Change readiness assessment]() 
+* [Change strategy]() 
+* [Change plan]() 
+* [Benefits register]() 
+* [Training and skills analysis]() 
+* [Closure letter]() 
+
+## Enabling Services
```
Comment (zedzedzed) [inline program/engagement/transition.md:55]: Extra line break needed

File: program/engagement/adoptionandevaluation.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (zedzedzed) [inline program/engagement/adoptionandevaluation.md:1]: Filename suggestion: `adoption-evaluation.md` or `adoption-and-evaluation.md`

File: program/engagement/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+## What does ‘Protected Utility’ mean?
```
Comment (zedzedzed) [inline program/engagement/FAQ.md:1]: Filename should be lowercase: `faq.md`

File: program/engagement/adoptionandevaluation.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+--- 
+layout: engagement 
+title: Adoption and Evaluation
+timeline: 
+- title: Step 1 – Enact continual service improvements
+  body: | 
+        The customer puts in place continual service improvements to their modern desktop suite. They may do this through continuing a support agreement with a technology partner, or their own in-house ICT and change capability.
```
[the remark this answers is not in the document]
Comment (patrickmorgan2) [reply program/engagement/adoptionandevaluation.md]: Hmm I can't quite see where the leading spaces are. I'll revisit this one later.

File: program/engagement/transition.md (hunk 1 above)
Comment (zedzedzed) [reply program/engagement/transition.md:62]: Not between bullets, I meant there needs to be a blank line before the start of the bullet list, eg: 

```
## Enabling Services

* The community...
```

Comment (patrickmorgan2) [reply program/engagement/transition.md:62]: OIC! I'll update that now

File: program/engagement/adoptionandevaluation.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (zedzedzed) [reply program/engagement/adoptionandevaluation.md]: There are extra spaces in front of `The customer` so it should be:

```
  body: | 
    The customer puts in place ...
```

**Comment by zedzedzed** [comment]
LGTM

**Pull request merged**